releasenotes/notes/remove-deprecated-items-2.0-89eac06472c59554.yaml

```diff
@@ -0,0 +1,92 @@
+---
+upgrade:
```

Comment:
This could probably use another pass once the change is finalized to make sure it includes everything